### PR TITLE
[Ready] Adds ERP

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

Adds a new emergency response team, the ERP! It stands for Emergency Response Party
the ERP includes:
-the coordinator: basically the captain of the operation, armed with a birthday cake, balloon and lots of fireworks to start the celebration
-the bouncer: the big boy, has a telebaton and an armor vest (if youre a loser you get a police hat too)
-the constructor: has tools, materials to build the party, disco tiles and an ethereal disco ball planter
-the cleaning service: your normal jannie, with his trusty mop, trash bag and cleaning grenades
-the comedian: a clown, armed with instruments, a bananium shield and chameleon suit/hat, for some comedy at the party
heres how the team looks
image

it also fixes some dumb things in ert code like the id cards being capitalized, unneeded copies of stuff, janitor and chaplain ert not having a hud icon and also adds a normal clown ert member (but hes not in any team so rest safe)
image
also bonus image: all ert members
image

## Why It's Good For The Game

With the new prs giving more celebration stuff, really seems like a missed opportunity to not have any party crew for admins to send
the new erp members seem like good for light hearted fun ert members, can be used for some admin events too
also adds clown ert before anyone else can

## Changelog
:cl:
add: Added ERP
fix: champion belt sprite now shows again
fix: centcom commanders and officials have mindshield implants
imageadd: a couple of ert related sprites have been added
tweak: ert members start with hailers on instead of them being in backpack
/:cl:

_pls dont close from antag frez_

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
where is the pusher

**Comment by Fikou** [comment]
> where is the pusher

the clown stabbed him to death

**Comment by trollbreeder** [comment]
Hey, you got the wrong branch Mr. Fikou, the april fools branch is 3 blocks down.

**Comment by Fikou** [comment]
> Hey, you got the wrong branch Mr. Fikou, the april fools branch is 3 blocks down.

fool there is no april fools branch

**Comment by trollbreeder** [comment]
> > Hey, you got the wrong branch Mr. Fikou, the april fools branch is 3 blocks down.
> 
> fool there is no april fools branch

Then you're 3 months too early.

**Comment by TheVekter** [comment]
Okay, the title got me. Well done.

> 
> 
> Hey, you got the wrong branch Mr. Fikou, the april fools branch is 3 blocks down.

Fuck you, leather man.

**Comment by HomocidlePancakes** [comment]
ERP deployed to dorm 2.

**Comment by carlarctg** [comment]
i don't like how their uniform is just repurposed clothing. it looks dumb

**Comment by itseasytosee** [comment]
The coordinator needs a new look and i dislike the sec hud on the bouncer

File: code/modules/antagonists/ert/ert.dm [hunk 1]
```diff
@@ -11,6 +11,7 @@
 	var/role = "Security Officer"
 	var/list/name_source
 	var/random_names = TRUE
+	var/rip_and_tear = FALSE
```
Comment (optimumtact) [inline code/modules/antagonists/ert/ert.dm]: this should come from the ert team definition I think

Comment (Fikou) [reply code/modules/antagonists/ert/ert.dm]: it shows the fluff text to the ert members separately so i thought that would be a better way to do it

**Comment by RiskySikh** [comment]
fikou made me comment this

**Comment by tralezab** [comment]
cool beans

**Comment by carshalash** [comment]
Where is the man in a spiderman costume?

**Comment by dee-dubya** [comment]
how do I speedmerge a PR when I don't have write access

**Comment by Bawhoppen** [comment]
I think they all need unique clothing before it should be merged... especially the one in the captain's outfit, since that is a special uniform that should be reserved for captains.

**Comment by SweptWasTaken** [comment]
Make the bouncer dress up like the cop from ymca

**Comment by trollbreeder** [comment]
BZZT, stand by for official transmission from Gas All Lizards Base-72.
"Send in the clowns..."
Transmission ends, have a secure day.
The Commander sat up from his seat. They called him Northey. But today, they will call him a hero.
[Central Command] Commander Northey yells, "Initiate protocol hotel oscar november kilo. Today is the day, boys!"
He slams the big red emergency party button, which sits right next to the emergency self destruct button. Floppy shoes, barefoot steps, and HONKs are heard behind his wall. He smiles with a shit eating grin, reaching for another cigar. 
TO BE CONTINUED.

**Comment by LilPenpusher** [comment]
"Mime just absorbed Clown. CentCom, we need ERP stat."

**Comment by ArcaneMusic** [comment]
"Hey sir?"
"What is it private."
"Why has the clown budget tripled since last quarter?"
"Do you want me to halve your salary instead?"
"...My mistake sir."

**Comment by Fikou** [comment]
i need to make some cool outfits for some of the guys

**Comment by Fikou** [comment]
ok done

**Comment by trollbreeder** [comment]
speedmerge now

**Comment by trollbreeder** [comment]
we did it gamers

**Pull request merged**